### ISSUE TITLE
Error compiling plugins/macro/macro.pl.

### ISSUE THREAD

**Issue by astroty** [OpenKore/openkore#855]
```Plugin contains syntax errors:
The encoding pragma is no longer supported at plugins/macro/Macro/Parser.pm line 5.
BEGIN failed--compilation aborted at plugins/macro/Macro/Parser.pm line 5.
Compilation failed in require at plugins/macro/Macro/Script.pm line 13.
BEGIN failed--compilation aborted at plugins/macro/Macro/Script.pm line 13.
Compilation failed in require at /home/pedro/Compilador/openkore-master/plugins/macro/macro.pl line 24.
BEGIN failed--compilation aborted at /home/pedro/Compilador/openkore-master/plugins/macro/macro.pl line 24.

Benchmark results saved to benchmark-results.txt
[raiseSkill] Plugin unloading or reloading
map unloaded.
*** OpenKore what-will-become-2.1 ( version ctime:2017_08_06 ) - Custom Ragnarok Online client ***
***   http://www.openkore.com/   ***

```
Perl version: perl 5, version 26, subversion 0 (v5.26.0) built for x86_64-linux-thread-multi
Python2: 2.7.13

**Comment by astroty** [comment]
How to fix: At the file 'plugins/macro/Macro/Parser.pm'
change:
'use encoding 'utf8';'
to
'use open ':std', ':utf8';'

**Comment by kaliwanagan** [comment]
Fixed by #856 thanks @astroty :)
